lighthouse-core/test/audits/service-worker-test.js

```diff
@@ -21,19 +21,6 @@ const assert = require('assert');
 /* eslint-env mocha */
 
 describe('Offline: Service Worker audit', () => {
-  it('reports driver error when given no Service Worker versions', () => {
```

Comment:
More or less, if I understand what you're asking. Audits never receive an error artifact; [runner produces an error audit result](https://github.com/GoogleChrome/lighthouse/blob/81b891d491e95135f1f5daf4278e1e27c93ac510/lighthouse-core/runner.js#L182) for them if any required artifacts are in error. That's tested by the test you linked.